x/ccv/provider/keeper/proposal.go

```diff
@@ -94,6 +95,20 @@ func (k Keeper) CreateConsumerClient(ctx sdk.Context, chainID string,
 	if lockUbdOnTimeout {
 		k.SetLockUnbondingOnTimeout(ctx, chainID)
 	}
+
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			clienttypes.EventTypeCreateClient,
```

Comment:
This should reference the new `EventTypeConsumerClientCreated` type, correct?

Reply:
Changed.